app/controllers/api/admin_controller.rb

```diff
@@ -0,0 +1,77 @@
+class Api::AdminController < Api::ApplicationController
+  before_action :admin_user
+
+  def view_profile
+    @user = User.find(params[:id])
+    render json: @user, serializer: UserSerializer
```

Comment:
please leave an empty space before render for better readability
this applies to all the methods